pages/docs/integrations/grouparoo-salesforce.mdx

```diff
@@ -1,193 +1,270 @@
 
-```js
-options: {
-  recordObject: "Contact",
-  recordMatchField: "Email",
-  groupObject: "Campaign",
-  groupNameField: "Name",
-  membershipObject: "CampaignMember",
-  membershipRecordField: "ContactId",
-  membershipGroupField: "CampaignId",
-  recordReferenceField: "AccountId",
-  recordReferenceObject: "Account",
-  recordReferenceMatchField: "Name",
-},
-```
+#### Optional Properties
 
-### Property Mappings
+In addition to the required email field, companies can add many Reserved or Custom Fields to sync with Grouparoo Properties. These can be accessed from the Send Record Property dropdown menu.
 
-These mappings are what tell Grouparoo which Properties you want to sync to the Destination and what they should be called. For this Plugin, **Grouparoo Properties** can be mapped to a **Salesforce Object** specified on the `options` object.
+### Group Data
 
-You can also map to custom attributes that you've defined on Salesforce by adding them directly to the Mapping object (also include the \_\_c suffix).
+Some Destination types support further grouping data to create remote lists or tags. This Destination does not offer that functionality.
 
-Here's an example of how this would look in the config file. Note that the keys refer to Salesforce Attribute names and values refer to Grouparoo Property IDs.
+## Salesforce Export Objects
+
+This Destination enables you to sync your Grouparoo Records as Objects in Salesforce.
+
+Companies use Grouparoo to keep their Salesforce Objects up-to-date in order to manage client relations effectively.
+
+To [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+- **Sync**: Add, update, and remove Salesforce Objects as needed.This option looks for new records, changes to existing records, and deletions.
+- **Additive**: Add and update Salesforce Objects as needed, but do not remove anybody. This option looks for new Objects to add to Salesforce and changes to existing Salesforce Objects, but does not keep track of deletions.
+- **Enrich**: Only update those Objects that already exist in Salesforce. Do not add or remove Objects. This option will only update existing Objects in Salesforce.
+
+### Destination Options
 
-```js
-mapping: {
-  Email: "email",
-  FistName: "firstName",
-  "custom_lifetime_value__c": "ltv",
-},
-```
+Some types of Destinations require further clarification about how data should sync. In the case of Salesforce Export Object Destinations, there are two required options to configure before moving on to mapping Record Data: **Record Object** and **Record Match Field.** This tells Grouparoo which Account field will be used to match up Salesforce Contacts with Grouparoo Records with a unique identifier. Depending on the Objects you have set up in Salesforce, there are other optional fields to use in setting up your Destination Options
```

Comment:
Curious why we don't have a screenshot for this config like we do for the others!